tests/robustness/validate/validate.go

```diff
@@ -31,10 +32,10 @@ import (
 func ValidateAndReturnVisualize(t *testing.T, lg *zap.Logger, cfg Config, reports []report.ClientReport, persistedRequests []model.EtcdRequest, timeout time.Duration) Results {
 	err := checkValidationAssumptions(reports, persistedRequests)
 	require.NoErrorf(t, err, "Broken validation assumptions")
-	linearizableOperations := patchLinearizableOperations(reports, persistedRequests)
-	serializableOperations := filterSerializableOperations(reports)
+	linearizableOperations, serializableOperations := prepareAndCategorizeOperations(reports)
+	patchedLinearizableOperations := patchLinearizableOperations(linearizableOperations, reports, persistedRequests)
```

Comment:
@serathius  Is passing in *operations with patched return time* and *original report* as arguments intended?

I am not super confident in some parts of the report patching code yet, so please double check the work here :( 

Thank you!

Reply:
Adding this to the comment